mappings/net/minecraft/container/GenericContainer.mapping

```diff
@@ -4,5 +4,21 @@ CLASS awz net/minecraft/container/GenericContainer
 	METHOD <init> (Laxt;ILaum;Lagk;I)V
 		ARG 2 syncId
 		ARG 5 rows
+	METHOD a generic9x1 (ILaum;)Lawz;
```

Comment:
Shouldn't these be named `createGeneric9x1`, to respect method conventions?

Reply:
I thought I would keep them short and closer to the pre-update names but now that they are methods I agree that they should follow the conventions for them. Will push a new commit shortly.